pkgs/applications/misc/lutris/default.nix

```diff
@@ -42,6 +42,7 @@ in buildPythonApplication rec {
 
   nativeBuildInputs = [ wrapGAppsHook ];
   buildInputs = [
+    gtk3 glib
```

Comment:
I'm seeing this is `buildPythonApplication`.
You need to use `strictDeps = false` for the setup hook to work, since it's default on in this deriver.

Reply:
Aha, that was in the documentation you gave me, wasn't it. With that all the variables for gtk3 and glib show up as well as one for gobject-introspection. Thanks.